babel.config.js

```diff
@@ -10,7 +10,7 @@ module.exports = (api) => {
         "@babel/preset-env",
         {
           targets: {
-            node: "12.13.0",
+            node: "14.16.0",
```

Comment:
I think it should be `14.15.0` (first LTS in v14) or `14.18.0` (ESM stabilization), but because we are still in commonjs (currently there are still many code in commonjs), so let's use `14.15.0` and postpone ESM migration on future (for the next major release)

Reply:
Yeah, makes sense. I will update.

Reply:
@alexander-akait Done.